counsel.el

```diff
@@ -4078,7 +4078,7 @@ Obeys `widen-automatically', which see."
             (cons (copy-marker (mark-marker)) marks)))
          (candidates (counsel-mark--get-candidates marks)))
     (if candidates
-        (counsel-mark--ivy-read candidates 'counsel-mark-ring)
+        (counsel-mark--ivy-read "Mark:" candidates 'counsel-mark-ring)
```

Comment:
Missing space after the prompt.

Reply:
Fixed. Please let me know if you find any other issues :smile:.